source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -324,9 +325,14 @@
                                 return valueItem;
                             }),
                             comments: _.map(commentsByKeyCollection, function (commentsByKey) {
-                                var commentItem = GetCommentItem(commentsByKey.commentRow.calificationId, commentsByKey[item.Key], commentsByKey.commentRow.evaluatorEmployee, commentsByKey.commentRow.evaluatorEmployee == self.userLogged)
+                                var commentItem = GetCommentItem(
+                                    commentsByKey.calificationColumn.calificationId,
+                                    commentsByKey[item.Key],
+                                    commentsByKey.calificationColumn.evaluatorEmployee,
+                                    commentsByKey.calificationColumn.evaluatorEmployee == self.userLogged)
 
                                 commentItem.IsEditingComment = ko.observable(false);
+                                commentItem.isEditable = self.isValueEditable(commentsByKey);
```

Comment:
I am not sure to understand it, `isValueEditable()` expects a _calification_ and _commentsByKey_ seems to be a dictionary.